cinema/views.py

```diff
@@ -127,6 +127,31 @@ def upload_image(self, request, pk=None):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                name="title",
+                description="Filter sessions by title (partial match)",
+                required=False,
+                type=str
+            ),
+            OpenApiParameter(
+                name="genres",
+                description="Filter sessions by movie genres (partial match)",
+                required=False,
+                type=str
+            ),
+            OpenApiParameter(
+                name="actors",
+                description="Filter sessions by movie actors (partial match)",
+                required=False,
+                type=str
+            )
+        ]
+    )
+    def list(self, request, *args, **kwargs):
+        return super().list(request)
```

Comment:
The `list` method should pass `*args` and `**kwargs` to `super().list(request, *args, **kwargs)` to ensure all arguments are correctly handled.